pkg/build/api/types.go

```diff
@@ -8,7 +8,10 @@ import (
 )
 
 // BuildLabel is the key of a Pod label whose value is the Name of a Build which is run.
-const BuildLabel = "build"
+const (
+	BuildAnnotation = "build"
```

Comment:
Follow @ironcladlou's examples in his most recent pulls.  Namespaced is required going forward.